supervisor/jobs_api_test.go

```diff
@@ -524,7 +525,38 @@ var _ = Describe("/v1/jobs API", func() {
 					"agent"           : "10.11.22.33:4455"
 				}
 			]`))
-		Ω(res.Code).Should(Equal(200))
+			Ω(res.Code).Should(Equal(200))
+		})
+	})
+
+	Context("With exact matching", func() {
+		It("should not retrieve jobs with only partially matching names", func() {
+			res := GET(API, "/v1/jobs?name=Redis+Daily+Backup&exact=t")
```

Comment:
Does this really test the "exact" nature of the query?  You almost need another fixture named `Redis Daily Backup - Extra Stuff`...